spy_integration_test.go

```diff
@@ -9,24 +9,34 @@ import (
 )
 
 func TestExecuteIntegration(t *testing.T) {
-	var buff bytes.Buffer
-	want := "echo Hello\nHello\n"
-	err := spy.Execute(&buff, "echo Hello")
+	want := struct{ a, b string }{
+		a: "Hello\n",
+		b: "************ START ************\nCOMMAND: echo Hello\n\nOUTPUT:\nHello\n",
+	}
+	var buff, buff2 bytes.Buffer
+	err := spy.Execute(&buff, &buff2, "echo Hello")
 
 	if err != nil {
 		t.Error(err)
 	}
+	got := struct{ a, b string }{
+		a: buff.String(),
+		b: buff2.String(),
+	}
 
-	got := buff.String()
-	if want != got {
-		t.Errorf("Want: %q, got: %q", want, got)
+	if want.a != got.a {
```

Comment:
If we use `cmp.Equal/Diff` to compare these long strings, it'll be much easier to see any tiny differences.